services/ui-src/src/components/pages/Export/ExportedReportPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { Helmet } from "react-helmet";
+import { Box, Center, Heading, Spinner, Flex } from "@chakra-ui/react";
+import { useStore } from "utils";
+import qmVerbiage from "verbiage/export/qm-export";
+import {
+  FormPageTemplate,
+  MeasurePageTemplate,
+  PageType,
+  ParentPageTemplate,
+  Report,
+} from "../../../types";
```

Comment:
Can we import `from "types";` here? I think types has the barrel file.

Reply:
you are correct, we can directly call it